cmd/run/flags.go

```diff
@@ -182,4 +182,13 @@ func bindRunFlags(command *cobra.Command) {
 	flags.Uint32("listObjects-max-results", defaultConfig.ListObjectsMaxResults, "the maximum results to return in ListObjects responses")
 	util.MustBindPFlag("listObjectsMaxResults", flags.Lookup("listObjects-max-results"))
 	util.MustBindEnv("listObjectsMaxResults", "OPENFGA_LIST_OBJECTS_MAX_RESULTS", "OPENFGA_LISTOBJECTSMAXRESULTS")
+
+	flags.Bool("allow-writing-deprecated-models", defaultConfig.AllowWritingDeprecatedModels, "allow writing of deprecated schemas")
+	util.MustBindPFlag("allowWritingDeprecatedModels", flags.Lookup("allow-writing-deprecated-models"))
+	util.MustBindEnv("allowWritingDeprecatedModels", "OPENFGA_ALLOW_WRITING_DEPRECATED_MODELS")
+
+	flags.Bool("allow-evaluating-deprecated-models", defaultConfig.AllowEvaluatingDeprecatedModels, "allow evaluating of deprecated schemas")
```

Comment:
`allow-evaluating-deprecated-models` is too ambiguous IMO. what if in the future we have 1.2 and we want to deprecate 1.1?

Reply:
used allow-writing-1.0-models and allow-evaluating-1.0-models